src/main/java/io/candydoc/ddd/tactical_pattern/DomainService.java

```diff
@@ -0,0 +1,27 @@
+package io.candydoc.ddd.tactical_pattern;
+
+import io.candydoc.ddd.model.DDDPattern;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class DomainService extends TacticalModel {
+
+  private static final Set<DDDPattern> ALLOWED_DEPENDENCIES =
```

Comment:
A domain service can depend on everything from the tactical view. It's like an orchestrator of business rules. So it might use commands, aggregates, events, repositories, factories.

The domain service should have its own PR